bookmarks/utils.py

```diff
@@ -128,10 +129,16 @@ def redirect_with_query(request, redirect_url):
     return HttpResponseRedirect(redirect_url)
 
 
-def generate_username(email):
+def generate_username(email, claims):
```

Comment:
for the record, the lib documentation is not up to date with the code, this function can have 2 args 
https://github.com/mozilla/mozilla-django-oidc/blob/main/mozilla_django_oidc/auth.py#L120